test/exec-tests.js

```diff
@@ -67,6 +69,7 @@ describe('Execution tests', () => {
             it('handles failing commands', () => {
                 return exec.execute('false', [], {}).then(testExecOutput).should.eventually.deep.equals({
                     code: 1,
+                    languageId: 'undefined',
```

Comment:
Making this field optional would mean we could "just" leave it off in all these cases.